ftw/linkchecker/command/checking_links.py

```diff
@@ -147,25 +155,49 @@ def iter_schemata_for_protal_type(portal_type):
 
 
 def create_and_send_mailreport_to_plone_site_responible_person(
-        email_address, site_info, broken_internal, broken_external):
-    path_to_report = create_excel_report_and_return_filepath(
-        site_info, broken_internal, broken_external)
+        email_address, broken_internal, broken_external):
+    path_to_report = create_excel_report_and_return_filepath(broken_internal,
+                                                             broken_external)
     send_mail_with_excel_report_attached(email_address, path_to_report)
 
 
-def create_excel_report_and_return_filepath(
-        site_info, broken_internal, broken_external):
-    # this function needs to create an excel report, safe it into a temp folder
-    # it then returns the path to that file.
-    # It makes use of the report generator from branch
-    # mo/excel_report_generator
-    pass
+def create_excel_report_and_return_filepath(broken_internal, broken_external):
+    filename = 'linkchecker_report_{}.xlsx'.format(
+        time.strftime('%Y_%b_%d_%H%M%S', time.gmtime()))
+    current_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Please use python's `tmpfile` modul, this way it generates files somewhere in the egg (cache) directory. 
We cannot necessarily write in those directories.

Reply:
I opened issue #29 to address this so we can move on here.